src/components/editor/Editor.tsx

```diff
@@ -294,7 +299,7 @@ class Editor extends React.Component<
           className={styles.close}
           onClick={this.onCancelClick}
         >
-          <i className="remove icon" />
+          <i className="remove icon"/>
```

Comment:
Maybe we should configure tslint to define a style for stuff like this (and below)? (unless it already is configured and I'm not aware of it 😛)